bin/worker_dev

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+QUEUE=* VERBOSE=1 RESQUE_PRE_SHUTDOWN_TIMEOUT=10 RESQUE_TERM_TIMEOUT=10 bundle exec rake environment resque:work
```

Comment:
Nice, this is super useful. I always wind up consulting `Procfile` to remember what to run. 👍

Reply:
🎆